packages/transaction-manager/test/unit/utils/test-data.ts

```
@@ -97,3 +161,45 @@ export const id3DecryptionProvider: DecryptionProviderTypes.IDecryptionProvider
+  private storedRawData;
+
+  constructor() {
+    this.storedRawData = {};
+  }
+
+  public async decrypt(
+    encryptedData: string,
+    options: {
+      encryptionParams: EncryptionTypes.IEncryptionParameters[];
+    },
+  ): Promise<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)